### PR TITLE
Construction of binary tree from preorder and inorder traversal

### PR DESCRIPTION
**Pull request by aaadddiii**
issue #321

### CODE REVIEW

File: Tree/readme.md [hunk 1]
```diff
@@ -49,5 +49,6 @@ that, this property should be satisfied at every node in the tree.
 ## Questions :
 * Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp)
```
Comment (raksha009) [inline Tree/readme.md:51]: plz remove this extra line.

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Review by raksha009 — APPROVED** [review]
LGTM!

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
link the issue to your PR. Also use class instead of struct in C++. Also add the comments to explain the steps.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
also there should be atleast 2 testcases along with the space complexity

**Comment by tarun26091999** [comment]
you have to link your issue with the PR not your PR with the PR itself. Also resolve the conflict of the branch.

**Review by tarun26091999 — APPROVED** [review]

**Pull request merged**